examples/BME280/BME280.ino

```diff
@@ -123,7 +123,23 @@ void setup() {
   digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
   adapter = new WebThingAdapter("weathersensor", WiFi.localIP());
 
+  // Set unit for temperature to "celsius". Other option "fahrenheit"
   weatherTemp.unit = "celsius";
```

Comment:
This should actually be “degree celsius”.

Reply:
This still needs to be changed.

Reply:
Changed "celsius" to "degree celsius".